src/components/OptionsSelector/BaseOptionsSelector.js

```diff
@@ -125,8 +125,11 @@ class BaseOptionsSelector extends Component {
             // Unregister the shortcut before registering a new one to avoid lingering shortcut listener
             this.unSubscribeFromKeyboardShortcut();
             if (this.props.isFocused) {
+                this.subscribeActiveElement();
```

Comment:
why is this needed?

Reply:
To resubscribe when the screen is refocused, otherwise, active element listener won't work anymore.